internal/eval/geo/geo.go

```diff
@@ -13,6 +13,9 @@ const earthRadius float64 = 6372797.560856
 // Bit precision for geohash - picked up to match redis
 const bitPrecision = 52
 
+// Bit precision for geohash string - picked up to match redis
+const bitPrecisionString = 10
```

Comment:
Redis uses 11, but the last char is set to `0`. We can use that to maintain compatibility or live with 10 which still has the same precision.

Reply:
Sounds good.